src/main/java/com/google/step/model/Tag.java

```diff
@@ -0,0 +1,11 @@
+package com.google.step.model;
+
+public class Tag {
+  public final long id;
+  public final String tag;
```

Comment:
Maybe call this field `tagName` or equivalent. A field `String tag` inside `class Tag` is confusing.